test/index.test.ts

```diff
@@ -1,14 +1,17 @@
 import os from 'os';
 import { TempDir } from './util';
-import { cosmiconfig, Options, LoaderSync } from '../src';
-import { createExplorer } from '../src/createExplorer';
-import * as loaders from '../src/loaders';
+import { LoaderSync } from '../src';
+import * as createExplorer from '../src/createExplorer';
+import * as createExplorerSync from '../src/createExplorerSync';
+import { loaders } from '../src/loaders';
 
-// @ts-ignore
-const createExplorerMock: typeof createExplorer & jest.Mock = createExplorer;
+let createExplorerMock = jest.spyOn(createExplorer, 'createExplorer');
+let createExplorerSyncMock = jest.spyOn(
+  createExplorerSync,
+  'createExplorerSync',
+);
```

Comment:
Does `jest.mock` not work to accomplish the equivalent?

Reply:
`jest.mock` removes the implementation of the file, while `jest.spyOn` does not (by default) which then tests the application instead of the mock we created.